### PR TITLE
acquisition: item quantities display

### PR DESCRIPTION
**Pull request by zannkukai**
* Updates item counter into the order detail view.
* Adds display of item ordered counter and item received counter into
  the order summary component.
* Fix CSS about `btn-show-more` class if included into a card section.

Co-authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: projects/admin/src/app/acquisition/api/acq-order-api.service.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class AcqOrderApiService {
    */
   getOrder(orderPid: string): Observable<AcqOrder> {
     const apiUrl = `/api/${this.resourceName}/${orderPid}`;
-    return this._http.get<any>(apiUrl).pipe(
+    return this._http.get<any>(apiUrl, {headers: { Accept: 'application/rero+json' }}).pipe(
```
Comment (Garfield-fr) [inline projects/admin/src/app/acquisition/api/acq-order-api.service.ts:63]: Extend your class with `BaseApi` and use `reroJsonheaders`

File: projects/admin/src/app/acquisition/components/order/order-brief-view/order-brief-view.component.html [hunk 2]
```diff
@@ -32,8 +32,9 @@ <h5 class="m-0"><a [routerLink]="[detailUrl.link]">{{ record.metadata.reference
   </div>
   <div class="col-3 ordinal-infos" *ngIf="record.metadata.total_amount > 0">
     <div class="total-amount">{{ record.metadata.total_amount | currency : record.metadata.currency }}</div>
-    <div class="items-counter" *ngIf="itemsCounter > 0">
-      {{ itemsCounter }} {{ itemsCounter | i18nPlural: {'=1': 'item', 'other': 'items'} }}
-    </div>
+    <ul class="item-quantity">
+      <li class="ordered-counter">{{ record.metadata.item_quantity.ordered}}</li>
```
Comment (Garfield-fr) [inline projects/admin/src/app/acquisition/components/order/order-brief-view/order-brief-view.component.html:36]: Missing space before `}`

File: projects/admin/src/app/acquisition/components/order/order-brief-view/order-brief-view.component.html [hunk 3]
```diff
@@ -32,8 +32,9 @@ <h5 class="m-0"><a [routerLink]="[detailUrl.link]">{{ record.metadata.reference
...
+      <li class="received-counter" *ngIf="record.metadata.item_quantity.received > 0">{{ record.metadata.item_quantity.received}}</li>
```
Comment (Garfield-fr) [inline projects/admin/src/app/acquisition/components/order/order-brief-view/order-brief-view.component.html:37]: Missing space before }

File: projects/admin/src/app/acquisition/components/order/order-summary/order-summary.component.html [hunk 4]
```diff
@@ -40,10 +40,14 @@
   </div>
   <!-- ORDER TOTAL AMOUNT ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
   <div class="col-4 pr-0">
-    <div class="border h-100 rounded bg-light">
+    <div class="border h-100 rounded bg-light position-relative">
       <div class="total-amount">
         <label>{{ order.total_amount || 0 | currency:order.currency:'symbol' }}</label>
       </div>
+      <ul class="item-quantity">
+        <li class="ordered-counter">{{ order.item_quantity.ordered}}</li>
```
Comment (Garfield-fr) [inline projects/admin/src/app/acquisition/components/order/order-summary/order-summary.component.html:48]: Missing space before }

File: projects/admin/src/app/acquisition/components/order/order-summary/order-summary.component.html [hunk 5]
```diff
@@ -40,10 +40,14 @@
...
+        <li class="received-counter" *ngIf="order.status !== acqOrderStatus.PENDING">{{ order.item_quantity.received}}</li>
```
Comment (Garfield-fr) [inline projects/admin/src/app/acquisition/components/order/order-summary/order-summary.component.html:49]: Missing space before }

File: projects/admin/src/app/acquisition/components/order/order-summary/order-summary.component.ts [hunk 6]
```diff
@@ -16,17 +16,55 @@
...
+import { AcqOrderApiService } from '../../../api/acq-order-api.service';
+import { AcqOrder, AcqOrderLine, AcqOrderLineStatus, AcqOrderStatus } from '../../../classes/order';
 
 @Component({
   selector: 'admin-order-summary',
   templateUrl: './order-summary.component.html',
   styleUrls: ['./order-summary.component.scss']
 })
-export class OrderSummaryComponent {
+export class OrderSummaryComponent implements OnInit, OnDestroy{
```
Comment (Garfield-fr) [inline projects/admin/src/app/acquisition/components/order/order-summary/order-summary.component.ts:29]: space before `{`

**Review by Garfield-fr — APPROVED** [review]

File: projects/admin/src/app/acquisition/api/acq-order-api.service.ts (hunk 1 above)
Comment (zannkukai) [reply projects/admin/src/app/acquisition/api/acq-order-api.service.ts:63]: It's not yet longer used... I corrected

**Pull request merged**